### PR TITLE
RIVM-vacamole results round 1

### PR DESCRIPTION
**Pull request by kylieainslie**
Results from the RIVM-vacamole model for round 1 scenarios. Only weekly cases are included.

### CODE REVIEW

**Comment by Bisaloo** [comment]
Hi @kylieainslie, thanks for your submission! Are you okay with me pushing directly to your branch to fix the tests? Please let me know what works best for you.

Otherwise, if you want to do it yourself, some changes I see for now:

- Run `git checkout main renv.lock` and `git checkout main renv/activate.R` and commit the result
- Rename the metadata file to `metadata-RIVM-vacamole.yml`
- Round the values from your model to integers

**Comment by kylieainslie** [comment]
@Bisaloo I'm happy for you to make any changes necessary directly. Thanks!

**Comment by Bisaloo** [comment]
@kylieainslie, we should be almost good now. The only change I didn't feel confident doing is:

- the `method` field in the metadata file must not have more than 200 characters.

This is because we use this short version of the methods to display it on the website. Could you write a shorter version of the methods please? Additional details can go into the `methods_long` section.

**Comment by kylieainslie** [comment]
@Bisaloo I've just committed a version of the metadata file with a shorter methods description in the `method` field.

**Comment by Bisaloo** [comment]
@kylieainslie, it looks like it's still too long :disappointed: 

``` r
nchar("We developed a deterministic, age-structured SEIR model with vaccination, which accounts for differences in susceptibility/infectiousness by age, seasonality, contact patterns, modes of vaccine protection, and waning immunity.")
#> [1] 226
```

<sup>Created on 2022-06-21 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1.9000)</sup>

File: data-processed/RIVM-vacamole/metadata-RIVM-vacamole.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
+team_name: Dutch National Institute of Public Health and the Environment (RIVM)
+model_name: vacamole
+model_abbr: RIVM-vacamole
+model_contributors: Kylie Ainslie (RIVM) <kylie.ainslie@rivm.nl>
+website_url: https://github.com/kylieainslie/vacamole
+license: cc-by-4.0
+team_model_designation: primary
+methods: We developed a deterministic, age-structured SEIR model with vaccination, which accounts for differences in susceptibility/infectiousness by age, seasonality, contact patterns, modes of vaccine protection, and waning immunity.
```
Comment (Bisaloo) [inline data-processed/RIVM-vacamole/metadata-RIVM-vacamole.yml:8]:
```suggestion
methods: Deterministic, age-structured SEIR model, accounting for differences in susceptibility/infectiousness by age, seasonality, contact patterns, modes of vaccine protection, and waning immunity.
```

**Comment by Bisaloo** [comment]
Thanks!

**Pull request merged**